mlir/dialects/stablehlo.zig

```diff
@@ -1261,6 +1261,29 @@ pub fn stablehloVersionFromCompatibilityRequirement(requirement: c.MlirStablehlo
     return state.call(requirement);
 }
 
+pub fn stablehloGetSmallerVersion(version1: []const u8, version2: []const u8) []const u8 {
+    const state = struct {
+        var buf: [32]u8 = undefined;
+
+        fn call(v1: []const u8, v2: []const u8) []const u8 {
+            var stream = std.io.fixedBufferStream(&buf);
+            var context = .{ .writer = stream.writer() };
+            const WriterContext = @TypeOf(context);
```

Comment:
can you refactor this as a helper in mlir ?

Reply:
you are right, we should have a look together to address that over the full mlir.zig too